soapui-testserver-api/src/test/resources/test-recipes/request-with-base64-attachment.json

```diff
@@ -0,0 +1,17 @@
+{
+  "testSteps": [
+    {
+      "type": "REST Request",
+      "method": "GET",
+      "URI": "http://google.com/",
+      "attachments" : [
+        {
+          "contentType" : "text/plain",
+          "name" : "A Name",
+          "contentId" : "An id",
+          "content": "Q29udGVudA=="
```

Comment:
it would be nice if this doesn't have to be base64 encoded - could we add an "encoding" property which could either be "base64" or "none" (default) - to allow for inline text attachments? @veinSmart didn't you work on something similar for TestServer!?